package.json

```diff
@@ -13,13 +13,14 @@
     "README.md"
   ],
   "scripts": {
-    "test": "mocha --require ts-node/register 'test/**/*.spec.{ts,tsx}'",
-    "coverage": "nyc npm run test",
+    "test": "vitest run",
```

Comment:
@imndaiga yeah, left this as is for now since moving to ESM would blow up the PR, will tackle that on a separate task and we probably need to plan a strategy on how we want to migrate our projects to ESM, but I do agree that we should now be moving away from `commonjs` as much as possible. There is an env var you can set to tune out the deprecation notice (`VITE_CJS_IGNORE_WARNING`) but yeah, I decided not to sweep it under the rug so we won't miss it.